recipes/cppunit/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "CppUnit" %}
+{% set version = "1.13.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
This will just be what we save the file as. So it doesn't matter if it is lower or upper.